components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementAdminService.java

```diff
@@ -51,8 +51,13 @@
      */
     public void createApplication(ServiceProvider serviceProvider)
             throws IdentityApplicationManagementException {
-        applicationMgtService = ApplicationManagementService.getInstance();
-        applicationMgtService.createApplication(serviceProvider, getTenantDomain(), getUsername());
+        try {
+            applicationMgtService = ApplicationManagementService.getInstance();
+            applicationMgtService.createApplication(serviceProvider, getTenantDomain(), getUsername());
+        }catch(IdentityApplicationManagementException idpException){
+            log.error("Error while creating application for service provider :" + serviceProvider , idpException);
```

Comment:
Remove space before the colon (":") and add a space after the colon.

Ex:
`log.error("Error while creating application for service provider: " + serviceProvider , idpException);`